### PR TITLE
Refactor/organise src folder

### PR DESCRIPTION
**Pull request by aagbotemi**
I organised the src folder with proper modules and lib.rs file

### CODE REVIEW

**Review by goebb — COMMENTED** [review]
why the `select_coin` function is duplicated in lib.rs and selectcoin.rs?

**Comment by aagbotemi** [comment]
please review again

**Comment by NeoZ666** [comment]
Kindly pull and just add the bnb as well

**Comment by aagbotemi** [comment]
> add the bnb

I've added the bnb

File: src/selectcoin.rs [hunk 1]
```diff
@@ -0,0 +1,135 @@
+use crate::algorithms::bnb::select_coin_bnb;
```
Comment (jkciw) [inline src/selectcoin.rs:1]: BNB can be added into the existing `use crate` statement.

Comment (aagbotemi) [reply src/selectcoin.rs]: please heck again @jkciw

**Comment by delcin-raj** [comment]
Looks good to me, think we can merge during the sync call today

**Comment by aagbotemi** [comment]
pr has been updated for removal of is segwit field @delcin-raj

**Comment by delcin-raj** [comment]
There are clippy errors.
Ignore unused variable in srd.rs.

But remove the unused variables in util.rs

**Comment by aagbotemi** [comment]
they are fixed

**Review by delcin-raj — APPROVED** [review]
LGTM

**Comment by delcin-raj** [comment]
Your commits need to be signed.

**Comment by NeoZ666** [comment]
LGTM too

**Comment by aagbotemi** [comment]
commit signed @delcin-raj

**Pull request closed without merging**